thoth/solver/cli.py

```diff
@@ -81,6 +86,16 @@ def cli(ctx=None, verbose=0):
     default="https://pypi.org/simple",
     help="A comma separated list of Python indexes to be used when resolving version ranges.",
 )
+@click.option(
+    "--dependency-index",
```

Comment:
Should it be
```suggestion
    "--dependency-indexes",
```
If we have a list of indexes?

Reply:
I see your point, I was unsure about it as well. I followed the already existing ``--index`` option in this case.

Reply:
Makes sense, we can leave it as it is then :+1: